src/_pytest/mark/structures.py

```diff
@@ -37,6 +42,9 @@ def get_empty_parameterset_mark(config, argnames, func):
         mark = MARK_GEN.skip
     elif requested_mark == "xfail":
         mark = MARK_GEN.xfail(run=False)
+    elif requested_mark == "fail_at_parametrize":
+        f_name = func.__name__
```

Comment:
Please also add the line number and improve the message a bit:

```python
_, lineno = getfslineno(func) 
raise ParametrizeError("Empty parameter set during parametrization of '%s' at line %d" % (f_name, lineno + 1))
```

This produces to me:

```
Empty parameter set during parametrization of 'test' at line 4
```

Which I think reads better. What do you think?

Reply:
I don't have a strong opinion, as the test name is most often sufficient. However, that does not cost much so I will include the line number.